lib/config/presets/internal/workarounds.ts

```diff
@@ -118,7 +118,10 @@ export const presets: Record<string, Preset> = {
           'java-jre',
           'sapmachine',
         ],
-        matchPackagePatterns: ['^azul/zulu-openjdk'],
+        matchPackagePatterns: [
+          '^azul/zulu-openjdk',
```

Comment:
I know this is already merged, but shouldn't the azul and bellsoft packages be added to the `matchPackageNames` array above so that it follows the javaLTSVersions `allowedVersions` expression?

Reply:
The azul and bellsoft names are not exact matches, and I believe `matchPackageNames` needs exact matches:
> Use this field if you want to have one or more exact name matches in your package rule.

https://docs.renovatebot.com/configuration-options/#matchpackagenames

Reply:
The intent of this this javaLTSVersions preset when I originally created it was to keep updates only to LTS versions (ie: 8, 11, 17, 21) etc...

If we aren't adding the azul and bellsoft flavors to the allowedVersions range, then perhaps this is better suited to a new java-based preset

Reply:
> The intent of this this javaLTSVersions preset when I originally created it was to keep updates only to LTS versions (ie: 8, 11, 17, 21) etc...

Yes, that's what I intend with the azul and liberica images. Currently, Renovate tries to bump those image references to non-LTS versions (ex, `18`) - the intent is to only bump to LTS versions.

Reply:
Ah, in that case I think you'll need to duplicate the `allowedVersions` into your package rule

Reply:
It's the same package rule - can you please clarify what should be duplicate and where?

Reply:
Apologies @candrews - the diff was rendering strangely on my mobile.  Please ignore my comments and thanks for enhancing this workaround